docs/conf.py

```diff
@@ -28,7 +28,24 @@
 if gfortran_exe is None:
     raise RuntimeError("Couldn't find the fortran compiler!")
 
-for filename in glob.glob("../lib/hipfort/*.f"):
+for filename in glob.glob("../lib/hipfort/*.f90"):
+    path = Path(filename)
+    # -P is to disable embedding line information
+    subprocess.check_call(
+        [
+            gfortran_exe,
+            "-E",
+            "-cpp",
+            "-P",
+            "-DUSE_FPOINTER_INTERFACES=1",
+            "-UUSE_CUDA_NAMES",
+            str(path),
+            "-o",
+            str(preprocessed_out_dir / path.name),
+        ]
+    )
```

Comment:
If this section is a duplicate of the one below, you can use `*.[fF]90` to match both. With that said, I'm not sure that we need to do this for `*.f90` files, as the lowercase indicates that they don't need preprocessing. (Although, I don't think it hurts, and you might still have to copy them to the other directory anyway, so I'm not sure the distinction truly matters.)